linux/devdoc/uuid_linux_requirements.md

```diff
@@ -38,8 +38,28 @@ MOCKABLE_FUNCTION(, int, uuid_produce, UUID_T, destination);
 
 `uuid_produce` fills destination's bytes with a unique ID.
 
+If the system has `libuuid` available:
+
 **SRS_UUID_LINUX_02_001: [** If `destination` is `NULL` then `uuid_produce` shall fail and return a non-NULL value. **]**
 
 **SRS_UUID_LINUX_02_002: [** `uuid_produce` shall call `uuid_generate` to generate a `UUID`. **]**
 
-**SRS_UUID_LINUX_02_004: [** `uuid_produce` shall succeed and return 0. **]**
\ No newline at end of file
+**SRS_UUID_LINUX_02_004: [** `uuid_produce` shall succeed and return 0. **]**
+
+If the system only has OSSP uuid available:

```

Comment:
>stem only has OSSP uuid available: [](http://example.com/codeflow?start=9&length=34)

Reaching here I started to question what is this ossp uuid
libuuid is the since forever default as UUID lib on Linux.

libuuid is available with linux kernel in util-linux.
How come we can't use it and we have to bring in ossp uuid?

Reply:
Did all this work, and then I found the right package to install. :( 
`tdnf install util-linux-devel` on Mariner 2.0